hw/top_darjeeling/ip_autogen/alert_handler/dv/cov/alert_handler_cov.core

```diff
@@ -7,7 +7,7 @@ description: "ALERT_HANDLER cov bind files"
 filesets:
   files_dv:
     depend:
-      - lowrisc:opentitan:top_darjeeling_alert_handler_component:0.1  # import alert_pkg
+      - lowrisc:opentitan:top_darjeeling_alert_handler_component:0.1  # import alert_handler_pkg
```

Comment:
Nit: this could probably have been done as part of the alert_pkg to alert_handler_pkg change. But let's not do this unless there are other changes that need to be done.